### ISSUE TITLE
Few Small Ideas

### ISSUE THREAD

**Issue by veryard** [anchorcms/anchor-cms#1311]
### Summary

Hey, i've been having a play around with AnchorCMS and it's a good start for a CMS but lacks alot of UX features that would help make the product much more appealing to new / existing users.

I'm just going to list a few small ideas here and hopefully can get some feedback and I can work towards these and pushing to the `pre-1.0-devlop` branch, if that's okay.

1. Admin Redesign - it's nice and simple, but it's not responsive and some parts are confusing. (Maybe TailwindCSS or Bootstrap)
2. Admin - Auto add new pages to navigation
3. Admin - Basic Reporting Dashboard
4. Recaptcha integration or some form of captcha
5. Theme Options - Nothing over the top, just maybe a json file that allows theme developers to provide basic options for admins to change

Thanks,
Brad.

**Comment by daviddarnes** [comment]
Lovely stuff Brad, here's brief comments from me on these ideas:
1. Definitely, couldn't agree more. Just needs some TLC, not a complete makeover though, I think people like the simple form of it
2. Could be nice, not sure how much it would be used though?
3. That would be cool! What would it report though? Could it be a plugin?
4. Is this for logging into the admin interface? Would be nice to add more verification to login forms
5. Yes please! I'm sure this wouldn't be hard to achieve

**Comment by veryard** [comment]
Thanks for the feedback David!

1. I could easily replicate the current design in either CSS framework, I just think some sections of the admin area could benefit greatly with a redesign.
3. There isn't currently a plugin system is there? So I assume it'd be good to write maybe a hook class that'll allow plugins to hook into existing functions etc.
4. Could be on login, I was thinking for comments.

**Comment by daviddarnes** [comment]
I don't think we need to use a CSS framework since the design is so simple, plus we could avoid loading a large chunk of CSS. Maybe just a CSS reset like sanitize.css and we go from there?You're right though, lots of places for improvement, the simple form of it is screaming for some TLC on smaller screens.

There is an early stage of the plugins but it's something that has been in the works for ages. A Dashboard is needed so people can access the various areas of the admin quickly, it's annoying that they land straight on pages (or is it posts?) when they log in

I suppose comments could benefit from that, to be honest I'm not sure why we have comments, seems like a plugin to me. I've made couple of sites on with Anchor and none of them use the comments feature.

**Comment by veryard** [comment]
https://github.com/veryard/anchor-cms/tree/admin-redesign

I started by cleaning up the `extend` tab in the administration area, the sidebar follows through when adding or editing etc.

Here's a snip:
![image](https://user-images.githubusercontent.com/3941758/55532664-4b272e00-56e1-11e9-917e-7671f8272832.png)
